src/hokusai/hokusai.yml

```
@@ -47,9 +47,12 @@ jobs:
         default: 20m
     steps:
       - setup
+      - run:
+          name: Install AWS IAM Authenticator
+          command: curl -o aws-iam-authenticator https://amazon-eks.s3-us-west-2.amazonaws.com/1.13.7/2019-06-11/bin/linux/amd64/aws-iam-authenticator && chmod +x ./aws-iam-authenticator && mv aws-iam-authenticator /usr/local/bin/
       - run:
           name: Configure
-          command: hokusai configure --kubectl-version 1.10.7 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+          command: mkdir ~/.hokusai && curl -o ~/.hokusai/config.yml https://artsy-provisioning-public.s3.amazonaws.com/hokusai/config-ci.yml && hokusai configure
```

You can break this up to make it easier to read

```suggestion
          command: |
            mkdir ~/.hokusai 
            curl -o ~/.hokusai/config.yml https://artsy-provisioning-public.s3.amazonaws.com/hokusai/config-ci.yml
            hokusai configure
```

---

This actually looks a lot like the configure step below...

You could potentially pull these out into a [command](https://circleci.com/docs/2.0/reusing-config/#authoring-reusable-commands). It could have that takes a URL and everything else could be the same.